cloudigrade/util/aws/helper.py

```diff
@@ -122,6 +131,8 @@ def _verify_policy_action(session, action):
                 Attribute='createVolumePermission',
                 OperationType='add',
             )
+        elif action in cloudtrail_actions:
+            return True
```

Comment:
Maybe throw a warning/info/debug in here stating that we couldn't actually check?